### DISCUSSION TITLE
Preselect value

### DISCUSSION THREAD

**Discussion by HemboPagi** [directus/directus#13685] [Archive]
Hi

I have a table with two column: _option-class_ and _option-value_.
I have linked this table with some other tables which filter then content (Type) based on _option-class_. This works nicely. But, how can i pre-fill the _option-class_ M20 field when adding the new value? 

I would like to actually hide the possibility to choose the _option-class_ from M20 interface.
![Screenshot 2022-06-02 at 17 06 05](https://user-images.githubusercontent.com/95413906/171648118-f11cf69f-bc0e-47bd-8aa5-26762dda9310.png)

**Comment by rijkvanzanten** [comment]
You should be able to set the default value, just like you would for any other regular field 👍🏻

**Comment by HemboPagi** [comment]
Not sure …

What i need is the same filter appleid when i display values

So when i start to add new Text type, Option Class would be already set.

On 6. June 2022 at 21:48:09, Rijk van Zanten ***@***.***)
wrote:

You should be able to set the default value, just like you would for any
other regular field 👍🏻

—
Reply to this email directly, view it on GitHub
<https://github.com/directus/directus/discussions/13685#discussioncomment-2892746>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AWX6NEXAL32KWG6XO34LAOLVNZBWNANCNFSM5XVDQ4AQ>
.
You are receiving this because you authored the thread.Message ID:
***@***.***>
